packages/protocol/test/common/gascurrencywhitelist.ts

```diff
@@ -1,88 +1,75 @@
-    let feeCurrencyWhitelist: FeeCurrencyWhitelistInstance
-    let registry: RegistryInstance
-    let stableToken: StableTokenInstance
-    let mockSortedOracles: MockSortedOraclesInstance
+  let feeCurrencyWhitelist: FeeCurrencyWhitelistInstance
+  let registry: RegistryInstance
+  let mockStableToken: MockStableTokenInstance
+  let mockSortedOracles: MockSortedOraclesInstance
 
-    const nonOwner = accounts[1]
+  const nonOwner = accounts[1]
 
-    const decimals = 18
-    const SECONDS_IN_A_WEEK = 604800
-    const goldAmountForRate = new BigNumber('0x10000000000000000')
-    const stableAmountForRate = new BigNumber(2).times(goldAmountForRate)
-    beforeEach(async () => {
-        stableToken = await StableToken.new()
-        registry = await Registry.new()
+  const goldAmountForRate = new BigNumber('0x10000000000000000')
+  const stableAmountForRate = new BigNumber(2).times(goldAmountForRate)
+  beforeEach(async () => {
+    mockStableToken = await MockStableToken.new()
+    registry = await Registry.new()
+    // await registry.setAddressFor(CeloContractName.StableToken, mockStableToken.address)
 
-        await stableToken.initialize(
-            'Celo Dollar',
-            'cUSD',
-            decimals,
-            registry.address,
-            fixed1,
-            SECONDS_IN_A_WEEK,
-            [],
-            [],
-            'Exchange'
-        )
+    mockSortedOracles = await MockSortedOracles.new()
+    await registry.setAddressFor(CeloContractName.SortedOracles, mockSortedOracles.address)
+    feeCurrencyWhitelist = await FeeCurrencyWhitelist.new(true)
+    await feeCurrencyWhitelist.initialize(registry.address)
+  })
 
-        mockSortedOracles = await MockSortedOracles.new()
-        await registry.setAddressFor(CeloContractName.SortedOracles, mockSortedOracles.address)
-        feeCurrencyWhitelist = await FeeCurrencyWhitelist.new()
-        await feeCurrencyWhitelist.initialize(registry.address)
+  describe('#initialize()', () => {
+    it('should have set the owner', async () => {
+      const owner: string = await feeCurrencyWhitelist.owner()
+      assert.equal(owner, accounts[0])
     })
 
-    describe('#initialize()', () => {
-        it('should have set the owner', async () => {
-            const owner: string = await feeCurrencyWhitelist.owner()
-            assert.equal(owner, accounts[0])
-        })
-
-        it('should not be callable again', async () => {
-            await assertRevert(feeCurrencyWhitelist.initialize(registry.address))
-        })
+    it('should not be callable again', async () => {
+      await assertRevert(feeCurrencyWhitelist.initialize(registry.address))
     })
+  })
 
-    describe('#addToken()', () => {
-        it('should not allow to add a token with an invalid oracle price', async () => {
-            await assertRevert(feeCurrencyWhitelist.addToken(stableToken.address))
-        })
+  describe('#addToken() with an invalid oracle price', () => {
```

Comment:
Sorry this might seem like a nit, but our general convention is to have one top level `describe` for the function in question (in this case `addToken`) and then have nested describe's that express the different cases

Reply:
my bad sorry